projects/netgrif-components-core/src/lib/navigation/service/uri.service.spec.ts

```diff
@@ -42,7 +42,7 @@ describe('UriService', () => {
 
     it('should get root node', () => {
         const root = service.root;
-        expect(root.id).toEqual('root');
+        expect(root.path).toEqual('root');
```

Comment:
On versions 6.4.0+, path cannot be `root`. The root path is `/`

Reply:
This is a test, where the root node is mocked with path `root` in `MockUriResourceService`. I can change it to `/` if you want to.